system_setup/common/wsl_conf.py

```diff
@@ -143,18 +146,25 @@ def drop_if_exists(self, config_section, config_setting):
                 self.config.write(configfile)
 
     def update(self, config_section, config_setting, config_value):
+        log.debug("Updating config: %s %s %s",
+                  config_section, config_setting, config_value)
+        if not self.config.has_section(config_section):
+            self.config.add_section(config_section)
+        if isinstance(config_value, bool):
+            config_value = str(config_value).lower()
```

Comment:
I don’t see the link with dry-run here, shouldn’t that be in the previous PR rather?